src/Controller/InvoiceEntryController.php

```diff
@@ -92,6 +94,11 @@ public function edit(Request $request, Invoice $invoice, InvoiceEntry $invoiceEn
         }
 
         if (InvoiceEntryTypeEnum::WORKLOG == $invoiceEntry->getEntryType()) {
+            // @todo Should accounts also be handled for non-worklog entries?
```

Comment:
Yes I think it should

Reply:
Fixed in https://github.com/itk-dev/economics/pull/122/commits/043938b04851e4dc9ced902831c9a107fe1107b2.